app/models/task.rb

```diff
@@ -1,3 +1,19 @@
 class Task < ActiveRecord::Base
+  belongs_to :person
 
+  def change_complete_status(complete_request)
+    complete_request == "1" ? complete : uncomplete
+  end
+
+  def complete
+    update(completed_at: Time.now)
+  end
+
+  def uncomplete
+    update(completed_at: nil)
+  end
+
+  def owner
+    Person.find(self.person_id).name
+  end
```

Comment:
Whether or not this method is useful is a judgement call, if there is any chance the source of the name will change in the future (eg will `task.person.name` change at some point?), I'd make this method, otherwise I'd just call `task.person.name` directly.